### PR TITLE
2.users

### PR DESCRIPTION
**Pull request by ameyunaikgithub**

### CODE REVIEW

File: users/file.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const f = require('fs');
+
+const read = (err,files)=>{
+      let names = '';
+      const find= (e)=>{
```
Comment (allopez7) [inline users/file.js]: Change e to something more descriptive.

File: users/file.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+              if(e !=='amey'){
```
Comment (allopez7) [inline users/file.js]: make sure indentations match

File: users/file.js (hunk 1 above)
Comment (ameyunaikgithub) [reply users/file.js]: can e be element ?

**Review by seemcat — APPROVED** [review]

**Review by allopez7 — APPROVED** [review]
Good JOB!!!!!

File: users/file.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+const read = (err,files)=>{
+  let names = '';
+  const find= (element)=>{
+    if(element !=='amey'){
+      names = names + element + '\n';
+    }
+  }
+  files.forEach(find);
+  console.log(names);
```
Comment (songz) [inline users/file.js:11]: change this to write file. Create a new file using `fs.writeFile(path, data, function)`

**Pull request merged**